contracts/test/mocks/apps/disputable/AgreementMock.sol

```diff
@@ -18,6 +18,7 @@ contract AgreementMock is IAgreement {
 
     function newAction(uint256, bytes, address) external returns (uint256) {
         // do nothing
+        // Note
```

Comment:
I know it's a mock, but perhaps we should have explicit returns for all the functions with returns :)